aws-sdk-v2/src/main/java/io/micronaut/aws/sdk/v2/service/cloudwatchlogs/CloudwatchLogsClientFactory.java

```diff
@@ -45,9 +47,11 @@ public class CloudwatchLogsClientFactory extends AwsClientFactory<CloudWatchLogs
      *
      * @param credentialsProvider The credentials provider
      * @param regionProvider      The region provider
+     * @param context             The application context
      */
-    protected CloudwatchLogsClientFactory(AwsCredentialsProviderChain credentialsProvider, AwsRegionProviderChain regionProvider) {
-        super(credentialsProvider, regionProvider);
+    protected CloudwatchLogsClientFactory(AwsCredentialsProviderChain credentialsProvider, AwsRegionProviderChain regionProvider, ApplicationContext context) {
+        super(credentialsProvider, regionProvider, context.findBean(
+            AWSServiceConfiguration.class, Qualifiers.byName(CloudWatchLogsClient.SERVICE_NAME)).orElse(null));
     }
```

Comment:
I believe you can receive directly the `@Named(CloudWatchLogsClient.SERVICE_NAME) AWSServiceConfiguration` instead of the application context